.circleci/runBuild.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
```

Comment:
@makarandp0 Can it be `runbuild.js`? We don't use camel-case for filenames either.